static/test/javascripts/spec/common/commercial/commercial-features.spec.js

```diff
@@ -325,28 +325,16 @@ define(['helpers/injector', 'Promise'], function (Injector, Promise) {
                 // i.e. we want to show the adblock message instead
                 detect.adblockInUse = Promise.resolve(true);
                 features = new CommercialFeatures;
-                features.async.membershipMessages.then(function (flag) {
+                features.async.canDisplayMembershipEngagementBanner.then(function (flag) {
                     expect(flag).toBe(false);
                     done();
                 });
             });
 
-            it('Does display messages on mobile', function (done) {
-                detect.getBreakpoint = function () { return 'mobile'; };
-                features = new CommercialFeatures;
-                features.async.membershipMessages.then(function (flag) {
-                    expect(flag).toBe(true);
-                    done();
-                });
-            });
-
-            it('Does not display messages to existing members', function (done) {
+            it('Does not display messages to existing members because they are already giving us money', function () {
```

Comment:
I wonder if this should explicitly express that they are a paying member rather than "already giving us money", which could be confused for having already given a contribution, especially if we start limiting contribution ads for people who have already contributed in the future